api/v1alpha1/hivelocitymachine_types.go

```diff
@@ -57,6 +57,7 @@ type HivelocityMachineStatus struct {
 	Ready bool `json:"ready"`
 
 	// Addresses contains the devices's associated addresses.
+	// question: CAPD uses this: []clusterv1.MachineAddress `json:"addresses,omitempty"`
```

Comment:
Check the other providers and then we'll follow what the majority does!

Reply:
aws and docker use clusterv1.MachineAddress

gcp, hetzner, azure, digitalocean use corev1.NodeAddress

I leave it like this and remove the question.

Reply:
@janiskemper 
Stefan Büringer said we should use `clusterv1.MachineAddress`. So will change this.

See: https://kubernetes.slack.com/archives/C8TSNPY4T/p1677236741690279?thread_ts=1677234946.342759&cid=C8TSNPY4T

Should I create a PR for Hetzner, too?

Reply:
Yes you can!